### ISSUE TITLE
Unable to local BuildConfig on Windows

### ISSUE THREAD

**Issue by alex2069** [NextFaze/dev-fun#32]
```
  Caused by: java.util.regex.PatternSyntaxException: Illegal/unsupported escape sequence near index 6
  (.*)(\intermediates\classes\|\tmp\kotlin-classes\|\tmp\kapt3\classes\)(.*)\META-INF\services\.*
```
Paths are generated using `File.separator` - probably needs to be escaped on Windows.

**Comment by BenTilbrook** [comment]
Any chance of a new version with the fix?

**Comment by alex2069** [comment]
Fixed in 2.1.0
